.cmake-format.yaml

```diff
@@ -0,0 +1,69 @@
+parse:
```

Comment:
First time I heard about .cmake-format, that's neat! One related question: we're beginning to "clog up" the root directory with many dot-files related to CI infrastructure. Can we move them to '.github' as well?

Reply:
Putting them in `.github` is probably a bad idea, as the files in there have special meanings for GitHub - like `.github/CONTRIBUTING.md`. issue templates, pull request templates, actions, workflows, etc. But this one could go anywhere, such as in `tools`. The problem is that if you try to run `cmake-format` from the command line, it will pick this up only if it's in the main folder. However, I would expect this and all the other formatters to only be run by `pre-commit`, so that's not a big deal. The `.pre-commit-config.yaml` file needs to be in the main folder. I rather think the `.clang-format` file will need to be in the main folder (IIRC, `-style=file` is the parameter you run, which doesn't include a path).

These are all just dot files, so they only make the main GitHub page longer (which is still a little irritating, but a common issue, especially for Python).

Reply:
(My general rule is that I always use the default name/location if something expects it, so tools "just work" if someone tries to run them; but if there's not special meaning to a name/path, I put it in a folder to keep the page clean)